boxsdk/client/client.py

```diff
@@ -159,6 +160,17 @@ def collaboration(self, collab_id):
         """
         return self.translator.translate('collaboration')(session=self._session, object_id=collab_id)
 
+    def trash(self):
+        """
+        Initialize a :class:`Trash` object.
+
+        :return:
+            A :class:`Trash` object.
+        :rtype:
+            :class:`Trash`
+        """
+        return Trash(self._session)
```

Comment:
This should still be translated if possible, to allow users to register their own `Trash` subclass if they want.